cypress/integration/dryRunner/manaSpend.js

```diff
@@ -0,0 +1,34 @@
+import s from './selectors'
+
+const MANA_DECK_ID =
+  'NU4xLDVOMiw1TjMsNU4yMyw1TjQsNU41LDVONiw1TjYyLDVOMTQsNVcxMiw1VzEwLDRXMTk'
+
+describe('Dry-runner — Gain and Spend mana', () => {
+  it('should be possible to gain/spend mana', () => {
```

Comment:
Shall this be part of `basics.js`?

Reply:
No I think it is fine this way, there could be more mana testing in the future - like with FC or Dawnsparks